contrib/get-ethos-checker

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# utility function to download a file
+function download {
+  if [ -x "$(command -v wget)" ]; then
+    wget -c -O "$2" "$1"
+  elif [ -x "$(command -v curl)" ]; then
+    curl -L "$1" >"$2"
+  else
+    echo "Can't figure out how to download from web.  Please install wget or curl." >&2
+    exit 1
+  fi
+}
+
+CVC_DIR=$(dirname $(dirname "$0"))
+mkdir -p $CVC_DIR/deps
+pushd $CVC_DIR/deps
+
+BASE_DIR=`pwd`
+mkdir -p $BASE_DIR/tmp/
+
+##### EO
+EO_DIR="$BASE_DIR/alf-checker"
+mkdir -p $EO_DIR
+
+# download and unpack ethos
+ETHOS_VERSION="204968e8cdd884782335481e4e7e967dc5becad9"
+download "https://github.com/cvc5/alfc/archive/$ETHOS_VERSION.tar.gz" $BASE_DIR/tmp/ethos.tgz
+tar --strip 1 -xzf $BASE_DIR/tmp/ethos.tgz -C $EO_DIR
+
+# build and install Ethos
+pushd $EO_DIR
+mkdir -p build && cd build
+cmake -DCMAKE_INSTALL_PREFIX="$BASE_DIR" ..
+make -j$(nproc)
+mkdir -p $BASE_DIR/bin
+cp ./src/alfc $BASE_DIR/bin/ethos
```

Comment:
Is this still called `alfc`?

Reply:
Yes, I am doing this in stages to make the changes small: the commit references in this script is still for a version of the repo that builds `alfc`.  A followup PR to cvc5 (after this one) will update to a commit that builds `ethos` and then the signatures will be changed to the new `eo::` syntax.